paper-slider.html

```diff
@@ -375,12 +407,23 @@
     },
 
     _knobdown: function(event) {
-      event.detail.sourceEvent.preventDefault();
+
+      // blur this slider if the user holds down the knob
+      // since we are calling `preventDefault` at the end of this function,
+      // the browser won't do it automatically.
+
       this._expandKnob();
+
+      // set the focus manually because we will call prevent default at the end
+      this.focus();
```

Comment:
I changed it, but I don't think it will make any difference though. The reason is because `preventDefault` does nothing but just changing the `defaultPrevented` property of the event object. Then, once this function ended, it will be opt to the event handlers in the event chain to do something based on what `defaultPrevented` means at that level. Once it reaches the `document` or the end of the event chain, it will prevent user selection and updating the focus element.